script/CREATE3Script.sol

```diff
@@ -13,7 +13,20 @@ import {L2YnOFTAdapterUpgradeable} from "@/L2YnOFTAdapterUpgradeable.sol";
 //      It should be inherited by the script that deploys the contracts, it shouldn't be used directly or as a
 // library.
 abstract contract CREATE3Script {
-    ICREATE3Factory public constant CREATE3_FACTORY = ICREATE3Factory(0x3Ab34A5758F42080A536865aD3a7D35E92861418);
+    ICREATE3Factory private constant CREATE3_FACTORY = ICREATE3Factory(0x3Ab34A5758F42080A536865aD3a7D35E92861418);
+
+    function getCreate3Factory() public view returns (ICREATE3Factory) {
+        if (block.chainid == 6900) {
+            // Exception for Nibiru
+            return ICREATE3Factory(0x9fB182200b19A8b9c67D0f152DECEE74a179a57a);
```

Comment:
override is because this factory is built with shangai, nibiru is not cancun compatible

address changes as a result.